src/worker/lambda/handlerPuller.go

```diff
@@ -42,27 +42,26 @@ func Copy(src, dest string) error {
 		return err
 	}
 
-	
 	if info.IsDir() {
 		return filepath.WalkDir(src, func(path string, d fs.DirEntry, err error) error {
-		    if err != nil {
-			return err
-		    }
-
-		    relPath, err := filepath.Rel(src, path)
-		    if err != nil {
-			return err
-		    }
+			if err != nil {
```

Comment:
what are these changes about?  are you fixing some sort of bug?  If so, perhaps a separate PR makes sense